src/fetch/payouts.py

```diff
@@ -120,11 +125,17 @@ def from_series(cls, frame: Series) -> RewardAndPenaltyDatum:
             secondary_reward_eth=int(frame["secondary_reward_eth"]),
             secondary_reward_cow=int(frame["secondary_reward_cow"]),
             quote_reward_cow=int(frame["quote_reward_cow"]),
+            protocol_fee_eth=int(frame["protocol_fee_eth"]),
         )
 
     def total_outgoing_eth(self) -> int:
         """Total outgoing amount (including slippage) for the payout."""
-        return self.payment_eth + self.secondary_reward_eth + self.slippage_eth
```

Comment:
All changes to the payment should be removed.
Those changes only make sense if we additionally change slippage accounting to exclude protocol fees.